apps/total-typescript/src/components/exercise-overlay.tsx

```diff
@@ -114,10 +113,14 @@ const Actions = () => {
 
 const ExerciseOverlay = () => {
   const {lesson, module} = useMuxPlayer()
+  const router = useRouter()
+  const {data: stackblitz, status} = trpc.stackblitz.byExerciseSlug.useQuery({
+    slug: router.query.exercise as string,
+    type: lesson._type,
+  })
```

Comment:
the router gives us out exercise slug and then we can load stackblitz by type

this component isn't universally portable (stackblitz). We send `_type` so we can switch on `solution`